pyi.Dockerfile

```diff
@@ -31,37 +49,33 @@ COPY scripts/perf_build.sh .
 RUN ./perf_build.sh
 
 # phpspy
-# ubuntu:20.04
-FROM ubuntu@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93 as phpspy-builder
-RUN apt update && apt install -y git wget make gcc
+FROM ubuntu${PHPSPY_BUILDER_UBUNTU} as phpspy-builder
+RUN if [ $(uname -m) = "aarch64" ]; then exit 0; fi; apt update && apt install -y git wget make gcc
 COPY scripts/phpspy_build.sh .
 RUN ./phpspy_build.sh
 
 # async-profiler
-# requires CentOS 6, so the built DSO can be loaded into machines running with old glibc.
-# centos:6
-FROM centos@sha256:dec8f471302de43f4cfcf82f56d99a5227b5ea1aa6d02fa56344986e1f4610e7 AS async-profiler-builder
+FROM centos${AP_BUILDER_CENTOS} AS async-profiler-builder
 COPY scripts/async_profiler_env.sh .
 RUN ./async_profiler_env.sh
 COPY scripts/async_profiler_build.sh .
 RUN ./async_profiler_build.sh
 
-FROM golang@sha256:f7d3519759ba6988a2b73b5874b17c5958ac7d0aa48a8b1d84d66ef25fa345f1 AS burn-builder
+FROM golang${BURN_BUILDER_GOLANG} AS burn-builder
 
 COPY scripts/burn_build.sh .
 RUN ./burn_build.sh
 
 
-# Centos 7 image is used to grab an old version of `glibc` during `pyinstaller` bundling.
-# This will allow the executable to run on older versions of the kernel, eventually leading to the executable running on a wider range of machines.
-# centos:7
-FROM centos@sha256:0f4ec88e21daf75124b8a9e5ca03c37a5e937e0e108a255d890492430789b60e AS build-stage
+# bcc & gprofiler
+FROM centos${GPROFILER_BUILDER} AS build-stage
 
 # bcc part
 # TODO: copied from the main Dockerfile... but modified a lot. we'd want to share it some day.
 
-RUN yum install -y \
-    git \
+RUN yum install -y git
+
+RUN if [ $(uname -m) = "aarch64" ]; then exit 0; fi; yum install -y \
```

Comment:
Doesn't the ARM build need any of these packages?

Reply:
Nope, they're needed for PyPerf which we currently don't build for ARM

Reply:
So perhaps add a comment at the top of the file explaining why it's not installed